vmdk_plugin/E2E_Tests/neg_CreateTenantsWithSameName_test.go

```diff
@@ -0,0 +1,84 @@
+//See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This is a negative test to verify if we are able to create two tenants same name.
+// Test will ssh to the esx and  create a tenant.
+// After creating the tenant, test tries to again create a tenant with same name.
+// Expected behavior: Two tenants with same name should not exist.
+
+// Test assumes that SSH cert has been setup to enable password-less login
+
+package e2e_test
+
+import (
+	TestUtil "github.com/vmware/docker-volume-vsphere/vmdk_plugin/utils/test_util"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+)
+
+var (
+	esx              string = os.Getenv("ESX")
+	tenantName       string = ""
+	ADMIN_CLI_TENANT string = "/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant "
+)
+
+/*
+Test steps:
+1. Create a new tenant.
+   Command: /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant create --name=<tenantName>
+2. Create another tenant with same name.
+   Tenant creation will fail as we had already created tenant with same name
+3. Do a list of tenants and check if we have only one with same name  present.
+   Command: /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant ls
+*/
+func TestCreateTwoTenantsWithSameName(t *testing.T) {
```

Comment:
It should be clubbed into the existing negative testcase file rather than keep creating separate files. It is better to have all trivial negative tests into the same file.